cudax/include/cuda/experimental/__container/async_buffer.cuh

```diff
@@ -552,6 +552,8 @@ void __copy_cross_buffers(stream_ref __stream, _BufferTo& __to, const _BufferFro
     __stream.get());
 }
 
+_LIBCUDACXX_DETAIL_MAGIC_NS_BEGIN
```

Comment:
All of our namespace macros are of the form `_CCCL_BEGIN_NAMESPACE_MEOW` We should use the same here